frontend/src/app/app-routing.module.ts

```diff
@@ -41,6 +47,12 @@ const routes: Routes = [
         component: NotFoundComponent,
         pathMatch: 'full',
     },
+    {
+        path: 'forgot-password',
+        loadChildren: () =>
+            import('./modules/forgot-password/forgot-password.module').then((m) => m.ForgotPasswordModule),
+    },
```

Comment:
1. Why you duplicate it?
2. Move to auth module, so routes will be auth/forgot-password
3. Add guards

Reply:
2 and 3 are still relevant

Reply:
If you moved, remove from here...